scout/parse/variant/transcript.py

```diff
@@ -252,13 +252,15 @@ def get_sift_prediction(entry):
 
 def get_dbsnp_list(entry):
     """Get dbSNP -the NCBI database of genetic variation- data if present in entry."""
-    dbsnp_list = []
-    variant_ids = entry.get("EXISTING_VARIATION")
-
-    if variant_ids:
-        for variant_id in variant_ids.split("&"):
-            if variant_id.startswith("rs"):
-                dbsnp_list.append(variant_id)
+    dbsnp_list = set()
+    dbsnp_keys = ["EXISTING_VARIATION", "RS_DBSNP150", "RS_DBSNP"]
+
+    for key in dbsnp_keys:
```

Comment:
Wow looks more complicated now! Perhaps expand also docstring with an example of a complicated entry?

Reply:
Yes, and probably another parse test for a current VEP; the VEP97 plus VEP OldSkool+ ones we carry are not necessarily current.